drivers/i2s/i2s_ll_stm32.c

```diff
@@ -109,6 +109,34 @@ static int i2s_stm32_enable_clock(struct device *dev)
 #ifdef CONFIG_I2S_STM32_USE_PLLI2S_ENABLE
 #define PLLI2S_MAX_MS_TIME	1 /* PLLI2S lock time is 300us max */
 static u16_t plli2s_ms_count;
+
+static u32_t shift_pllr(u8_t pllr)
```

Comment:
Since CONFIG_I2S_STM32_PLLI2S_PLLR is defined at compile time, it would be legit use a pre-processor macro as done in clock controller and save some code:
```
#define _pllr(v) LL_RCC_PLLI2SR_DIV ## v
#define pllr(v) _pllr(v)
```

Reply:
ok, it seems really smart!